notion/manifest.json

```
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0a709534-658c-4d8f-99a3-566dd7cd809b",
+  "app_id": "notion",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor your Notion workspace events and customize detections in Datadog",
+    "title": "Notion",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Setting up the Datadog integration in Notion",
+        "image_url": "images/Notion_DD.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Notion event logs in Datadog",
+        "image_url": "images/Notion_DD_2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Event Management",
+      "Category::Collaboration",
+      "Supported OS::Any",
+      "Submitted Data Type::Events"
+    ]
+  },
+  "assets": {
+    "dashboards":
+    {"notion_events": "assets/dashboards/notion_events.json"}
```

Since this is an integration, we'll need to add the integration asset in here:
```suggestion
    {"notion_events": "assets/dashboards/notion_events.json"},
    "integration": {
      "source_type_name": "notion",
      "configuration": {},
      "events": {
        "creates_events": false
      },
      "service_checks": {
        "metadata_path": "assets/service_checks.json"
      }
    },
```
Please also add back the `service_checks.json` file under the `assets` folder. The contents of this file should just be an  empty array: `[]`